networkit/cpp/centrality/GroupClosenessLocalSwaps.cpp

```diff
@@ -0,0 +1,362 @@
+// networkit-format
+
+#include <algorithm>
+#include <cassert>
+#include <omp.h>
+#include <queue>
+
+#include <networkit/auxiliary/Random.hpp>
+#include <networkit/centrality/GroupClosenessLocalSwaps.hpp>
+
+namespace NetworKit {
+
+GroupClosenessLocalSwaps::GroupClosenessLocalSwaps(const Graph &G, const std::vector<node> &group,
+                                                   count maxSwaps)
+    : GroupClosenessLocalSwaps(G, group.begin(), group.end(), maxSwaps) {}
+
+void GroupClosenessLocalSwaps::init() {
+    const auto n = G->upperNodeIdBound();
+
+    distance.assign(n, 0);
+    gamma.assign(n * group.size(), false);
+    canSwap.assign(group.size(), false);
+    visited.assign(n, false);
+
+    idxMap.clear();
+    idxMap.reserve(group.size());
+
+    stack.assign(n, 0);
+    farness.assign(group.size(), 0);
+    farnessDecrement.assign(group.size(), 0);
```

Comment:
Decrement -> Decrease. Similarly, if that occurs elsewhere: Increment -> Increase.